src/index.html

```diff
@@ -22,6 +22,370 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__navigation navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a
+              class="navigation__link is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="navigation__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
```

Comment:
According to the BEM methodology and the checklist, elements of a block should be separated by two underscores, and modifiers should be separated from the block or element name by two dashes. In this case, the class 'card' should be a block, and elements inside should follow the pattern 'card__element-name'. For example, 'card__image', 'card__name', etc. Ensure all child elements of the 'card' block follow this naming convention.